packages/frontapp/src/app/components/Scanner/index.tsx

```diff
@@ -1,6 +1,6 @@
 import { ChangeEvent, FC, memo, useEffect, useMemo } from 'react';
 import { scanImageData } from 'zbar.wasm';
-import { BrowserMultiFormatOneDReader } from '@zxing/browser';
+// import { BrowserMultiFormatOneDReader } from '@zxing/browser';
```

Comment:
Uncomment this line. If the scanner package doesn't work properly, you can create a new issue and assign it to yourself